templates/common/azure/files/azure-chrony.yaml

```diff
@@ -0,0 +1,50 @@
+filesystem: "root"
```

Comment:
I'd remove `azure-` prefix as the folders are already namespaced

Reply:
sure, will remove it with next fixup!

Reply:
fixed